src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -250,18 +336,18 @@ public ZipArchiveEntry GetEntry(string entryName)
                 throw new NotSupportedException(SR.EntriesInCreateMode);
 
             EnsureCentralDirectoryRead();
-            ZipArchiveEntry result;
+            ZipArchiveEntry? result;
```

Comment:
Nit: You could've inlined this declaration into the `TryGet`. `_entriesDictionary.TryGetValue(entryName, out ZipArchiveEntry? result);`